exporter/otlpexporter/internal/arrow/stream.go

```diff
@@ -182,7 +216,9 @@ func (s *Stream) write(ctx context.Context) {
 
 		if err := s.client.Send(batch); err != nil {
 			// The error will be sent to errCh during cleanup for this stream.
-			s.telemetry.Logger.Error("arrow send", zap.Error(err))
+			if !errors.Is(err, io.EOF) && !errors.Is(err, context.Canceled) {
```

Comment:
same here, do we care about the other cases?

Reply:
(maybe there are no other cases we expect, regardless may be good to put in a log?)

Reply:
I added TODOs about reconsidering EOF and Canceled cases. Maybe if we have the max-connection-age feature enabled we will want to know more about canceled contexts.